lib/samson/form_builder.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module Samson
+  class FormBuilder < ActionView::Helpers::FormBuilder
+    def input(attribute, as: :text_field, help: false, label: false)
```

Comment:
I wonder if there's a more obvious name for this, something like `labelled_input` maybe?